schema/all-zones-requests.json

```diff
@@ -562,33 +562,12 @@
           "type": "object",
           "required": [
             "address",
-            "dns_servers",
-            "ntp_servers",
             "type"
           ],
           "properties": {
             "address": {
               "type": "string"
             },
-            "dns_servers": {
```

Comment:
To be clear: the change to this file is okay, even though it's an on-disk interface, because new schema describes a subset of the old schema?  i.e., we will be able to continue reading files that had these properties?

Reply:
Correct on both counts. If `OmicronZoneType` had a `#[serde(deny_unknown_fields)]` we'd have a problem, but I believe we only attach that attribute to structs that are deserialized from things that should never be wrong (e.g., config files that are shipped in the zone alongside the binary).